traits/07-betydb-sql-access.Rmd

```diff
@@ -2,11 +2,6 @@
 
 will be derived from https://github.com/pi4-uiuc/2017-bootcamp/blob/master/content/post/2017-05-30-databases-and-sql.Rmd#with
 
-## On workbench
-
-On the TERRA REF Workbench, you have access to the database. These connections will only work if you are on the workbench.
-
-
 ## Using PostgresSQL Studio
```

Comment:
@dlebauer I'm not sure what to replace this section with.

Reply:
dont need to replace;  sufficient to have the docker version of terraref bety